upi/openstack/01_security-groups.yaml

```diff
@@ -149,6 +149,14 @@
       port_range_min: 2379
       port_range_max: 2380
 
+  - name: 'Create master-sg rule "etcd" for pod network'
+    os_security_group_rule:
+      security_group: "{{ os_sg_master }}"
+      protocol: tcp
+      remote_ip_prefix: "{{ cluster_network_cidrs }}"
+      port_range_min: 2379
+      port_range_max: 2380
```

Comment:
do we need here too 
when: os_networking_type == "CiscoACI"

Reply:
we can do this as a follow up, but lets do that after we get confirmation from RH that its actually not required.